src/Routers/FeaturesRouter.js

```diff
@@ -20,10 +20,10 @@ export class FeaturesRouter extends PromiseRouter {
     if (!req.auth.isMaster) {
       return masterKeyRequiredResponse();
     }
-    
+
     return Promise.resolve({
       response: {
-        results: [getFeatures()]
+        results: getFeatures()
```

Comment:
Raw object without a nested layer - sounds great.
I am not sure whether `results` is a standard at this point, since it's used roughly for 2 endpoints only. Queries and schema. :)